64 expand u substitution

enorms opened the pull request:
Hey @heckman, I'm putting this in a PR so we have a focused place for discussion. If you don't like it for any reason I have no problem if you deny or delete this.

Since you made this public, and I'm so excited to bring collaborative version tracking features outside of the coding world, it's sort of a test to try working together on math code. There was some spacing I wanted to help with, for example, and get comfortable with everything so I could look for "sign errors." I also wanted to recommend putting the equations at the top of the page, rather than the bottom, because it will work better in a scrolling or non-paginated situation (though it is kind of pretty to have it on the bottom for print).

This particular change is an expansion of the integral of cos squared to explicitly show the u substitution. It may not strictly be necessary but it was good practice and on a time-pressure setting I expect it will be nice to have.

Technical note: could not build locally and found the repo would build in Overleaf so that's where I made the edits (on my fork)

enorms committed:
64 expand u substitution

heckman commented:
### Hello, I appreciate any help with this!

Between now and the final exam I'm resisting the urge
to make changes other than

- corrections to the math
- additional integrals

Anything else is much too dangerous, as I could spend zero time actually studying.

I'd be happy to work on this after the exam&mdash;I'm keen to keep practicing latex.

That being said, at this moment I'm going to avoid studying by addressing some of your comments.

### Spacing

Could you be more specific? and maybe make it an issue?
I'm pretty new to collaborating, but that might be a good place.

I'm aware that the spacing next to the equals signs is a bit wonky.
I think it has to do with the position of the `&`s.
I'd like to move them all to the left of the operators,
which I think should help,
but the switch might get messy,
so that will definitely happen post-exam.

### Footer

The 'titles' are in the footer because that's where I find them
easiest to flip through.

That being said, moving them to the header is trivial.
They are defined in `calcexam.cls`
by `\fancyfoot[RO,LE]{\circled\no}` and `\fancyfoot[C]{\theintegral}`.

Perhaps there could be an additional target for digital usage.
It could possibly be `html` rather than `pdf`.
Could this could also be made into an issue?

### Changes to I-64

Typically I've haven't done a formal substitution when the "chain" of the "chain rule" is a constant.

I'm fairly conservative with descriptive asides; when I use them I use a `\nb` macro defined towards the end of `calcexam.cls`. Its first argument is text, followed by an optional argument for math to be put in a align environment. (It has a couple other options too.)

For substitutions I use a `\sub` macro I've written a few similar macros to standardize formatting, and to abstract-out the english language. Unfortunately these macros are not well commented. Two other macros: `\parts` and `\sohcahtoa` are particularly useful.

### Thanks again for checking over the math

I'm open to other suggestions too, but I'm resisting making other changes, until after the final exam. (I really want to such as review the typeface and add a copyright page with permissive license.) 

If you're still interested in this project post-Calculus II, I'd be grateful for the collaboration. I could also help you get it working on your machine&mdash;I think we have similar setups (mactex/vscode/latexworkshop).

### Cheers!

enorms commented:
> Between now and the final exam I'm resisting the urge
> to make changes other than corrections to the math [and] additional integrals

Yeah that makes sense I can support that.


> That being said, at this moment I'm going to avoid studying by addressing some of your comments.

lol


> The 'titles' are in the footer because that's where I find them easiest to flip through.
> That being said, moving them to the header is trivial.

Interesting... the repo should match your preferences. Cool that it's easy, maybe there's a way to change certain things only locally / on my fork.


> Typically I've haven't done a formal substitution when the "chain" of the "chain rule" is a constant.
> I'm fairly conservative with descriptive asides...

Interesting. Yeah I find that what I want to do changes with my comfort. This repo should match your preferences.


 > ...I use ... macro[s]...

That's cool! I will need to look into that more (after the exam).


> Thanks again for checking over the math... I'm resisting making other changes, until after the final exam.

You're welcome and seems like a good idea! I'll keep any further comments focused on that.


> If you're still interested in this project post-Calculus II, I'd be grateful for the collaboration. I could also help you get it working on your machine—I think we have similar setups (mactex/vscode/latexworkshop).

Cool!
